assets/scss/bootstrap-migration/_custom.scss

```diff
@@ -1,7 +1,21 @@
-@import "custom/backgrounds";
-@import "custom/popover";
-@import "custom/navbar";
-@import "custom/sizing";
-@import "custom/buttons";
-@import "custom/type";
-@import "custom/modals";
+@import 'custom/backgrounds';
+@import 'custom/popover';
+@import 'custom/navbar';
+@import 'custom/sizing';
+@import 'custom/buttons';
+@import 'custom/type';
+@import 'custom/modals';
+@import 'custom/lists';
+
+.add-cart-item-total {
```

Comment:
Shouldn't it be a module, or at least moved to separate file?

Reply:
It's just a custom css, I couldn't think of what it can be attributed to. So i left it in core custom.scss file. It's too small of a change to be in the module in my opinion